docs/event.md

```diff
@@ -43,6 +43,29 @@ is fired. The following key values are also supported:
  - `%arguments` - The arguments passed when the event was dispatched/triggered.
 
 
+@signature `(SCOPE_KEY_LOOKUP SCOPE_EVENT)='CALL_EXPRESSION'`
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<my-component (. someevent)="doSomething()"/>
+```
+
+@param {can-stache/expressions/key-lookup} SCOPE_KEY_LOOKUP A key to specify what scope the SCOPE_EVENT is in.
```

Comment:
not really the "scope"  ... there's only one scope.  It's really what object's (or event emitters) events should be bound to.